### ISSUE TITLE
Rubik Hero Crashes my Computer

### ISSUE THREAD

**Issue by Pulpers859** [ValveSoftware/Dota2-Gameplay#12024]
### Description

For the last 6 games as Rubik, the game has crashed and froze my computer to where I have to shut it down. This does not happen with any other hero as I have played many games in between using Rubik.

**Comment by Dest1xxx** [comment]
)

**Comment by TobyFlendersonFromHR** [comment]
cya in low prio

**Comment by Hikki413s** [comment]
Does it crash like this bro ? Mine all started after I demo snapfire's new collector's cache set, when I casted her ulti the bug happened. Now I cant play dota without banning rubick or snapfire, which is kind of hard because we cant specifically ban them like in turbo.
![Bugs](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145239862/0623f5ad-917b-48c3-97e0-20dd37070828)

**Comment by Pulpers859** [comment]
Yes, the last time it crashed the screen looked like! That but I haven’t noticed it with Snapfire. I don’t think I play many games against that hero.

**Comment by Ryugamin** [comment]
Glad, I wasn't the only one but man... this update fuk me over now I literally have to play after successfully banning rubick, otherwise I'm being sent to lp

**Comment by mKoy1218** [comment]
I have the same issue.

**Comment by Qetuo22** [comment]
I have the same problem. I don't know how to fix this

**Comment by 444Raven** [comment]
why is valve still not giving a **** to this?

**Comment by arshunvalery** [comment]
I have the same issue with Rubick. Still not fixed
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/76811287/672bd7fc-815b-440c-b1af-b343032cb370)

**Comment by 444Raven** [comment]
how long will this take I cant play him after the graphics patch
